packaging/README.md.in

```diff
@@ -79,26 +84,23 @@ increase the track mapping parameter value +1 for each new audio object input pl
 5. Switch between the different renderings by exclusive-soloing (CMD+Alt+Click (MacOS) / Ctrl+Alt+Click (Win)) the monitoring tracks.
 
 ## How to use (long version)
-Have a look at this video where these points are explained:
-- Introduction
-- Installation
-- Tutorial 1: Understanding the EAR Production Suite Plugins
-- Tutorial 2: Creating an ADM Project from scratch with the EAR Production Suite Plugins
-- Tutorial 3: Using an Existing ADM File with the EAR Production Suite Plugins
-- Tutorial 4: Using Third-Party Spatial Audio Plugins with the ADM Export Source Plugin
-- Tutorial 5: Using Non-Object Audio with the ADM Export Source Plugin
+Have a look at this [video](https://youtu.be/u7P5mEFY76k) where these points are explained:
 
-[![Watch the video](https://irt-a.akamaihd.net/EAR-Production-Suite/Intro_image.png)](https://irt-a.akamaihd.net/EAR-Production-Suite/Intro_Beta-release.mp4)
+- [Introduction](https://youtu.be/u7P5mEFY76k)
+- [Installation](https://youtu.be/u7P5mEFY76k?t=81)
+- [Tutorial 1](https://youtu.be/u7P5mEFY76k?t=162): Understanding the EAR Production Suite Plugins
+- [Tutorial 2](https://youtu.be/u7P5mEFY76k?t=462): Creating an ADM Project from scratch with the EAR Production Suite Plugins
+- [Tutorial 3](https://youtu.be/u7P5mEFY76k?t=824): Using an Existing ADM File with the EAR Production Suite Plugins
+- [Tutorial 4](https://youtu.be/u7P5mEFY76k?t=1077): Using Third-Party Spatial Audio Plugins with the ADM Export Source Plugin
+- [Tutorial 5](https://youtu.be/u7P5mEFY76k?t=1418): Using Non-Object Audio with the ADM Export Source Plugin
 
-## Survey
-We would highly appreciate if you could fill in [this survey](https://docs.google.com/forms/d/e/1FAIpQLSfosvonWSOUt_-Q30MJtcGERs2XVqS2EaKauOzGsc3B4oWKIA/viewform). It takes only a few minutes and will help us to create a better tool based on your feedback.
+[Watch the video](https://irt-a.akamaihd.net/EAR-Production-Suite/Intro_Beta-release.mp4)
```

Comment:
We should point out the install directory thing - something like this; https://github.com/ebu/ear-production-suite/commit/de069fa9e80dc1cc524bc4a94be206464fe925c0